base/config/cockroach

```diff
@@ -0,0 +1,4 @@
+cockroach.host=cockroachdb-proxy
+cockroach.port=26257
+cockroach.seed.hosts=cockroachdb-0.cockroachdb,cockroachdb-1.cockroachdb,cockroachdb-2.cockroachdb
```

Comment:
As discussed, this will break existing clusters when upgrading. The clusters I'm aware of are setting their own configmap which one include some of these newer values; init-job you might get away with as it runs once, but you won't with the statefulset.

Not against this, but we'd need to PR existing clusters (there seems to be 37 from a simple grep) to add the new env vars. Once migrated to this SHA, we can just have everyone delete their configs and use the default.